processing/src/main/java/org/apache/druid/frame/read/columnar/StringFrameColumnReader.java

```diff
@@ -170,7 +155,7 @@ private void validate(final Memory region)
     }
   }
 
-  private static boolean isMultiValue(final Memory memory)
+  static boolean isMultiValue(final Memory memory)
```

Comment:
Should make this and other methods/variables  protected 
```suggestion
  protected static boolean isMultiValue(final Memory memory)
```